{{cookiecutter.project_name}}/.github/workflows/tests.yml

```diff
@@ -97,14 +97,15 @@ jobs:
 
       - name: Upload coverage data
         if: always() && matrix.session == 'tests'
-        uses: "actions/upload-artifact@v3"
+        uses: actions/upload-artifact@v4
         with:
-          name: coverage-data
+          name: coverage-data-${{"{{"}} matrix.python {{"}}"}}-${{"{{"}} matrix.os {{"}}"}}
```

Comment:
Would jinja's `{% raw %}` block be more readable here? Could you try using it on the entire value instead?